_scripts/banned_plugins.js

```diff
@@ -2,5 +2,7 @@ module.exports = [
 
   'chai-51st-state', // Banned for profane language
   'chai2.0-webdriver-promised', // Fork with poor metadata
+  'lodash-match-pattern', // not a real chai plugin, chai-match-pattern is the real one
+  'shimo-chai-as-promised', // dupe of chai-as-promised
```

Comment:
Opened https://github.com/shimohq/chai-as-promised/pull/1 for this. Same as above.